Cargo.toml

```diff
@@ -59,7 +59,7 @@ apollo-parser = "0.7"
 apollo-encoder = "0.8"
 
 # https://github.com/apollographql/federation-rs
-apollo-federation-types = "0.11.0"
+apollo-federation-types = { git = "https://github.com/apollographql/federation-rs.git", rev = "c4683c3c071ac02643d620f620371dc404b7a13f" }
 
```

Comment:
I remember you [pinning this to the PR](https://github.com/apollographql/federation-rs/pull/518) you have outstanding : did the federation-rs team approve that related PR yet 👀 ?